src/main/java/net/socialgamer/cah/data/Game.java

```diff
@@ -1448,9 +1448,15 @@ public ErrorCode playCard(final User user, final int cardId, final String cardTe
     final Player player = getPlayerForUser(user);
     if (player != null) {
       player.resetSkipCount();
+
       if (getJudge() == player || state != GameState.PLAYING) {
         return ErrorCode.NOT_YOUR_TURN;
       }
+
+      if (playedCards.getCards(player).size() >= blackCard.getPick()) {
```

Comment:
I haven't tried this yet, but I suspect this will throw an NPE the first time a player tries to play a card, since they will not have yet played a card and [getCards is documented to return null in that situation](https://github.com/ajanata/PretendYoureXyzzy/blob/master/src/main/java/net/socialgamer/cah/data/PlayerPlayedCardsTracker.java#L95).